web-next/app/google/sandbox-best-practices/page.module.css

```
@@ -1,654 +1,662 @@
+
+  /* Per-domain accent palette */
+  --c-agent: #7c9eff;
+  --c-api: #5eead4;
+  --c-container: #f2b84b;
+  --c-cpp: #fb7185;
+  --c-browser: #c4b5fd;
+
+  --code-bg: #0a1526;
+  --sidebar-w: 300px;
+  --radius: 14px;
+  --font-jp: var(--font-sans), 'Noto Sans JP', -apple-system, BlinkMacSystemFont, 'Hiragino Kaku Gothic ProN', 'Yu Gothic', sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)